src/app/modal/modal.component.scss

```diff
@@ -1,6 +1,6 @@
 @import "node_modules/govuk-frontend/govuk/all";
 
-.landing-page {
+.modal-page {
 
   & a {
     color: #1d70b8 !important;
```

Comment:
😖 something to deal with later but i will check how this got in, !important is nasty

Reply:
removed the css file